frontend/src/API/CoffeeChatAPI.ts

```diff
@@ -71,4 +71,18 @@ export default class CoffeeChatAPI {
       return result;
     });
   }
+
+  public static async getCategoryToMembers(semester: string): Promise<BingoBoard> {
+    const res = APIWrapper.get(`${backendURL}/bingo-board/${semester}`).then((res) => res.data);
+    return res.then((val) => {
+      if (val.error) {
```

Comment:
We could move this error handling to wherever its used in the UI. I think it should be the UI who gets to decide how to handle this. `getCategoryToMembers` should just return the promise (could be pending/fulfilled/rejected).

The I might format this function the same way the other ones above are, and just return the promise.